code/modules/reagents/chemistry/recipes.dm

```diff
@@ -250,6 +250,9 @@
 	var/power = modifier + round(created_volume/strengthdiv, 1)
 	if(power > 0)
 		var/turf/T = get_turf(holder.my_atom)
+		if(isorgan(holder.my_atom) && !T) // bandaid fix since get_turf doesn't work on organs in the body
+			var/obj/item/organ/curorgan = holder.my_atom
```

Comment:
I could probably codegolf this shit even harder if i tried
`if (!T && var/obj/item/organ/curorgan = holder.my_atom)`

Reply:
does this facilitate reactions within organs?

Reply:
They already did have reactions. This fixes explosions not actually occuring; it caused runtime errors because get_turf didn't work on organs in the body.

Reply:
oh, works for me i guess

couldnt validate that this is actually true but it seems like a reasonable thing to merge

i also like the idea of people exploding due to a chemical cocktail in their blood